lib/setup/Openst.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Web3 = require('web3');
+
+const TokenHolder = require('./../ContractInteract/TokenHolder');
+const GnosisSafe = require('./../ContractInteract/GnosisSafe');
+const Recovery = require('./../ContractInteract/Recovery');
+const UserWalletFactory = require('./../ContractInteract/UserWalletFactory');
+const ProxyFactory = require('./../ContractInteract/ProxyFactory');
+const CreateAndAddModules = require('./../ContractInteract/CreateAndAddModules');
+
+/**
+ * Performs deployment of openst contracts.
+ */
+class OpenST {
```

Comment:
I think this class is redundant, it can just be a method.

Reply:
Do you think it's blocker or good to have? 
The same has been integrated in JLP openst setup.

Reply:
I have added above in the ticket.

Reply:
made the change for alignment with other repos 👍